components/molecules/m-cookie-notification.vue

```diff
@@ -4,7 +4,7 @@
       <div class="cookie">
         <div class="cookie__message">
           {{ message }}
-          <a 
```

Comment:
this file has just a space, can you undo the changes?

Reply:
Sure.